### ISSUE TITLE
KeyError on calendar.timegm(pokestops[f['id']]['last_modified']

### ISSUE THREAD

**Issue by halsafar** [PokemonGoMap/PokemonGo-Map#1452]
```
Traceback (most recent call last):
  File "PokemonGo-Map/pogom/search.py", line 510, in search_worker_thread
    parsed = parse_map(args, response_dict, step_location, dbq, whq, api)
  File "PokemonGo-Map/pogom/models.py", line 882, in parse_map
    'last_modified': calendar.timegm(pokestops[f['id']]['last_modified'].timetuple()),
KeyError: u'be4cd69ee9774ae1a208c8141eaa6f74.16'
```

Getting this all the time since updating with the PRs from Sept 30 to Oct 3.  Not sure which one.  I cannot go backwards since one of them marked the DB for upgrade and now older checkouts won't work unless I nuke my DB.

Temporary bad fix is to just set last_modified: 0.  Clearly not a fix but I am scanning again.

I'll also note this traceback causes the program to tell the user their account is banned even though it isn't.

**Comment by Darkknightz** [comment]
This is fixed in #1444 which was merged earlier. If you pull again it should be fixed

**Comment by halsafar** [comment]
Confirmed.
